src/main/java/Primitives.java

```diff
@@ -1,17 +1,128 @@
+        System.out.println("The distance between the numbers is: " + ((number1 - number2) < 0 ? -(number1 - number2) : (number1 - number2)));
+        System.out.println("The biggest number is: " + Math.max(number1, number2));
+        System.out.println("The smallest number is: " + Math.min(number1, number2));
+        System.out.println("");
+    }
+
+        //Practice with chars - convert string to chars
+    public void getChars(String string) {
+        System.out.println("Exercise 4. - convert string to chars and print on different lines");
+        for (int n = 0; n < string.length(); n++) {
+            System.out.println(string.charAt(n));
+        }
+        System.out.println("");
     }
 
-        //Practice with float and double
-            //Write a Java program that reads a number in inches, converts it to meters.
-            //Note: One inch is 0.0254 meter.
+    public void countChars(String string) {
+        int letter = 0;
+        int num = 0;
+        int space = 0;
+        int other = 0;
+        for(int i = 0; i < string.length(); i++){
+            if(Character.isLetter(string.charAt(i))){
+                letter ++;
+            }
+            else if(Character.isDigit(string.charAt(i))){
+                num ++;
+            }
+            else if(Character.isSpaceChar(string.charAt(i))){
+                space ++;
+            }
+            else{
+                other ++;
+            }
+        }
+        System.out.println("Exercise 5. - count characters");
+        System.out.println("Your string has: ");
+        System.out.println("letters: " + letter);
+        System.out.println("spaces: " + space);
+        System.out.println("numbers: " + num);
+        System.out.println("other: " + other);
+        System.out.println("");
+    }
 
-        //Practice with char
+    public void getLetters(String string) {
+        String temporaryString = "";
+        for (int n = 0; n < string.length(); n++) {
+            if(Character.isLetter(string.charAt(n))) {
+                temporaryString = temporaryString + string.charAt(n);
+            }
+        }
+        System.out.println("Exercise 6. - extract letters only");
+        System.out.println(temporaryString);
+        System.out.println("");
+    }
 
         //Practice with booleans
+    public void countOddAndEvenNumbers(int[] numArray) {
+        int evenNumbers = 0;
+        int oddNumbers = 0;
+        for(int i = 0; i < numArray.length; i++) {
+            if(numArray[i] % 2 == 0)
+            {
+                evenNumbers++;
+            }
+            else
+                oddNumbers++;
+        }
+        System.out.println("Exercise 7. - count odd numbers and even numbers in array");
+        System.out.println("Number of even elements in the array: " + evenNumbers);
+        System.out.println("Number of odd elements in the array: " + oddNumbers);
+        System.out.println("");
+    }
+
 
+    Random rand = new Random();
```

Comment:
From what I could gather, 
you don't use Random class anywhere. 

Why keep it ?

Reply:
True that.